kvmagent/kvmagent/test/vm_plugin_testsuite/test_vm_dump.py

```
@@ -0,0 +1,38 @@
+from kvmagent.plugins import vm_plugin
+from kvmagent.test.utils import vm_utils, network_utils, pytest_utils
+from kvmagent.test.utils.stub import *
```

Ruff (0.11.9)

`from kvmagent.test.utils.stub import *` used; unable to detect undefined names

(F403)